apps/tenant-management-webapp/src/app/pages/admin/services/form/definitions/addRemoveResourceTagModal.tsx

```diff
@@ -156,8 +156,12 @@ export const AddRemoveResourceTagModal: FunctionComponent<AddRemoveResourceTagMo
               } as ResourceTag);
 
               if (!tagAlreadyAdded()) {
-                dispatch(clearAllTags());
-                dispatch(fetchAllTags());
+                setTimeout(() => {
+                  dispatch(clearAllTags());
+                }, 300);
+                setTimeout(() => {
+                  dispatch(fetchAllTags());
+                }, 300);
```

Comment:
Hi, Thang: in this case might be put dispatch into setInterval,  which used to repeatedly dispatch clearAllTags and fetchAllTags every 300 milliseconds until the interval is cleared.

const intervalId = setInterval(() => {
                    dispatch(clearAllTags());
                   dispatch(fetchAllTags());
                  clearInterval(intervalId);
               }, 300);